src/main/resources/assets/iris/lang/nl_nl.json

```diff
@@ -1,5 +1,19 @@
 {
   "iris.shaders.reloaded": "Shaders opgeladen!",
   "iris.keybind.reload": "Herladen Shaders",
-  "iris.shaders.reloaded.failure": "Mislukt om Shaders te herladen! Reden: %s"
+  "iris.shaders.reloaded.failure": "Mislukt om Shaders te herladen! Reden: %s",
+
+  "options.iris.apply": "Solliciteer",
+  "options.iris.refresh": "Vernieuwen",
+  "options.iris.openShaderPackFolder": "Open de map Shader Pack...",
+  "options.iris.refreshShaderPacks": "Vernieuw Shader Pack",
+  "options.iris.shaderPackSelection.addedPack": "Shader Pack toegevoegd \"%s\"",
+  "options.iris.shaderPackSelection.addedPacks": "Toegevoegd %s Shader Packs",
+  "options.iris.shaderPackSelection.failedAdd": "Bestanden waren niet geldig Shader Packs",
+  "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" is geen geldig Shader Pack",
+  "options.iris.shaderPackSelection.copyError": "Kon geen Shader Pack toevoegen \"%s\"",
+  "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" staat al in uw Shader Packs map!",
+
+  "pack.iris.select.title": "Selecteer",
+  "pack.iris.list.label": "+ Shader Packs slepen en neerzetten om toe te voegen"
```

Comment:
The Dutch translation is pretty terrible. Here's my suggestion:

> "iris.shaders.reloaded": "Shaders herladen!",  
> "iris.keybind.reload": "Herlaadt Shaders",  
> "iris.shaders.reloaded.failure": "Shaders herladen is mislukt! Reden: %s",  
> "options.iris.apply": "Toepassen",  
> "options.iris.refresh": "Verversen",  
> "options.iris.openShaderPackFolder": "Open de Shader Pack map...",  
> "options.iris.refreshShaderPacks": "Ververs Shader Pack", 
> "options.iris.shaderPackSelection.addedPack": "Shader Pack \"%s\" toegevoegd",  
> "options.iris.shaderPackSelection.addedPacks": "%s Shader Packs toegevoegd",  
> "options.iris.shaderPackSelection.failedAdd": "Bestanden zijn geen geldige Shader Packs",  
> "options.iris.shaderPackSelection.failedAddSingle": "\"%s\" is geen geldige Shader Pack",  
> "options.iris.shaderPackSelection.copyError": "Kon Shader Pack \"%s\" niet toevoegen",  
> "options.iris.shaderPackSelection.copyErrorAlreadyExists": "\"%s\" staat al in uw Shader Packs map!",  
> "pack.iris.select.title": "Selecteren",  
> "pack.iris.list.label": "+ Voeg Shader Packs toe via drag-and-drop"

Or alternatively for that last one (if you want to avoid all English):

> "pack.iris.list.label": "+ Voeg Shader Packs toe door te slepen en neer te zetten"